packages/credentials/src/verifiable-credential.ts

```diff
@@ -278,8 +292,11 @@ export class VerifiableCredential {
     validatePayload(vcTyped);
 
     return {
+      /** The issuer of the VC. */
```

Comment:
These are already defined up above. Are the docs really complaining about every single key of a returned object? If so that probably needs to get fixed on the docs check

Reply:
Oh these are separate and unrelated to the class, nevermind.